README.md

```diff
@@ -49,6 +51,25 @@ The image is compatible with the following databases system :  `MySQL` (default)
 
 ## Quick start
 
+### Easy way : with docker-compose
+
+eXo Platform Community edition is based on sub-components :
+
+- Database (mysql or psql)
+- Elastic Search
+- Mongodb
+
+Theses sub-components are not provided in the docker image exoplatform/exo-community.
+
+If you want to run all components in one action, we provide
+a [guide](https://docs.exoplatform.org/en/latest/GettingStartedeXoCommunity.html) based on a docker-compose file. This will allow
+you to start all components needed by the platform.
+
+### Advanced way : with docker images
+
+If you prefer, you can run separatly each components by starting their own docker image. Then you have to use properties describe
```

Comment:
```suggestion
Alternatively, you may want to run each component separately with containers. To do this, you can use properties described
```

Reply:
it may be useful to link to the container images required so user can review their own README and manually configure them.

Reply:
Modification proposed